addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -13,6 +13,13 @@ This binding integrates the [Yeelight Lighting Product](https://www.yeelight.com
 Project will be updated to support future product.
 
 
+## Precondition
+To control Yeelight device with this binding, you need to connect the device to your
+local network at first with Yeelight app, the app could be downloaded from apple store
```

Comment:
The app could be downloaded from [Apple Store](https://itunes.apple.com/app/yeelight/id977125608) or [Google Play](https://play.google.com/store/apps/details?id=com.yeelight.cherry)
Please add appropriate links.